HISTORY.rst

```diff
@@ -59,7 +65,7 @@ Other Changes:
 * Update doc string for ``is_copy_only_path()`` according to pep257, thanks to
   `@mathagician`_ and `@terryjbates`_ (#935, #949)
 * Update doc strings in ``cookiecutter/extensions.py`` according to pep257,
-  thanks to `@meahow`_ (#996)
+  thanks to `@meahow`_ (#996, #742)
```

Comment:
You mention one issue in two places. 
Such commits are hard to review. It is not correct at all.
One mention of PR itself is enough to find all history of the issue.